### PR TITLE
Fixes #12369: Port userManagement to multiversionned technique

### PR DESCRIPTION
**Pull request by ncharles**
https://www.rudder-project.org/redmine/issues/12369

### CODE REVIEW

File: techniques/systemSettings/userManagement/userManagement/8.1/metadata.xml [hunk 1]
```diff
@@ -15,6 +15,7 @@ along with this program.  If not, see <http://www.gnu.org/licenses/>.
 -->
 
 <TECHNIQUE name="Users">
+  <DEPRECATED>This technique version has been superseded by a new version. It will no longer be available in the next stable version of Rudder. Please upgrade to the latest version.</DEPRECATED>
```
Comment (amousset) [inline techniques/systemSettings/userManagement/userManagement/8.1/metadata.xml]: not deprecated in 4.3

File: techniques/systemSettings/userManagement/userManagement/9.0/metadata.xml [hunk 2]
```diff
@@ -26,22 +27,23 @@ along with this program.  If not, see <http://www.gnu.org/licenses/>.
     <OS version=">= 2008">Windows</OS>
     <OS version=">= 5.3">AIX</OS>
     <AGENT version=">= 3.6.0">cfengine-community</AGENT>
+    <AGENT version=">= 4.2.0"></AGENT>
```
Comment (amousset) [inline techniques/systemSettings/userManagement/userManagement/9.0/metadata.xml]: why 4.2?

Comment (ncharles) [reply techniques/systemSettings/userManagement/userManagement/9.0/metadata.xml]: I forgot the dsc in it

**Comment by ncharles** [comment]
Commit modified

**Pull request merged**